sonar-java-plugin/src/main/java/org/sonar/plugins/java/JavaSquidSensor.java

```diff
@@ -96,10 +142,13 @@ public void execute(SensorContext context) {
     return StreamSupport.stream(inputFiles.spliterator(), false).map(InputFile::file).collect(Collectors.toList());
   }
 
+  private ExecutorService createExecutor() {
+    int numThreads = Runtime.getRuntime().availableProcessors() + 1;
+    return Executors.newFixedThreadPool(numThreads, new ThreadFactoryBuilder().setNameFormat("SonarJava-parallel-analysis-%d").build());
+  }
+
   private JavaVersion getJavaVersion() {
-    JavaVersion javaVersion = JavaVersionImpl.fromString(settings.getString(Java.SOURCE_VERSION));
-    LOG.info("Configured Java source version (" + Java.SOURCE_VERSION + "): " + javaVersion);
-    return javaVersion;
+   return JavaVersionImpl.fromString(settings.getString(Java.SOURCE_VERSION));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)